opm/simulators/wells/StandardWell.hpp

```diff
@@ -254,6 +254,25 @@ namespace Opm
                                     DeferredLogger& deferred_logger) const;
 
 
+        std::vector<double> getPrimaryVars() const override
+        {
+            const int num_pri_vars = this->primary_variables_.numWellEq();
```

Comment:
What's the typical value of `num_pri_vars`?   Something like 3-4?  How often do we expect to call `getPrimaryVars()`?  Once per well per non-linear iteration, or more frequently?  If we're calling the function more frequently, then we might also consider an out-parameter
```C++
mumble getPrimaryVars(std::vector<double>& privars) const override
```
to limit the number of (small) allocations.

Reply:
For debugging it may be called more often, but in production it would typically be once per non-linear iteration.

Reply:
Yes, that is correct, sorry for being unclear.